R/Nepal_SEES.R

```diff
@@ -0,0 +1,24 @@
+#'
+#'
+#' SEES Typhoid data
+#'
+#' A subset of data from the SEES project highlighting Typhoid longitudinal
+#' data from Nepal.
+#'
+#' @format ## `nepal_sees`
+#' A [base:data.frame()] with 904 rows and 8 columns:
```

Comment:
I think this change will fix 
![image](https://github.com/user-attachments/assets/199d4f30-01cc-44ca-a1ca-7c36c844bab9)


```suggestion
#' A [base::data.frame()] with 904 rows and 8 columns:
```

Reply:
Fixed issue